selfdrive/boardd/panda.h

```diff
@@ -0,0 +1,74 @@
+#pragma once
+
+#include <ctime>
+#include <cstdint>
+#include <pthread.h>
+
+#include <libusb-1.0/libusb.h>
+
+#include "cereal/gen/cpp/car.capnp.h"
+#include "cereal/gen/cpp/log.capnp.h"
+
+// double the FIFO size
+#define RECV_SIZE (0x1000)
+#define TIMEOUT 0
+
+// copied from panda/board/main.c
+struct __attribute__((packed)) health_t {
```

Comment:
maybe we should make this a standalone header panda and include it here?

Reply:
or we can wait until we move this abstraction into panda

Reply:
Let's do it when we move it to panda. I also want to create an enum for all the different commands instead of the hardcoded hex values in main.c and panda.cc

Reply:
We can do this later, but ideally there's a USB command to query board features. there's lists like this in a few different places in openpilot that should just be part of a panda abstraction